build.gradle

```diff
@@ -40,12 +43,14 @@ dependencies {
     implementation('com.sun.xml.bind:jaxb-core:2.3.0')
     implementation('com.fasterxml.jackson.datatype:jackson-datatype-hibernate5:2.10.0')
     implementation group: 'org.hibernate', name: 'hibernate-search-orm', version: '5.11.3.Final'
+    implementation group: 'org.hibernate', name: 'hibernate-spatial', version: '5.1.0.Final'
     implementation group: 'org.postgresql', name: 'postgresql', version: '42.1.4'
     implementation('com.google.maps:google-maps-services:0.10.1')
+    implementation('com.bedatadriven:jackson-datatype-jts:2.4')
+    implementation group: 'org.opengeo', name: 'geodb', version: '0.8'
```

Comment:
This dependency includes H2, so I remove it from below.

Reply:
Maybe put a comment in here about that